app/scripts/models/auth_brokers/base.js

```diff
@@ -271,6 +271,10 @@ define([
        */
       emailVerificationMarketingSnippet: true,
       /**
+       * Should the view try to synchronise sign-in state across tabs?
+       */
+      interTabSignIn: true,
```

Comment:
Part of the problem, I think, is that the we're modelling a capability of the view as a capability of the auth broker. If it was negated and called something like `disableInterTabSignIn`, I think it would sit better here. Thoughts?